dask_geopandas/morton_distance.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+import pandas as pd
+from dask_geopandas.hilbert_distance import _continuous_to_discrete_coords
```

Comment:
yes I agree, both `_continuous_to_discrete_coords` and it's dependency `_continuous_to_discrete` could be centralized (outside of hilbert_distance.py) 

My initial thought was that this didn't warrant a standalone script. However, both `_hilbert_distance` and `_morton_distance` also both use `_distances_from_coordinates` in [[1]](https://github.com/tastatham/dask-geopandas/blob/a1eae6ceb8cd580f3d1f8240d22e52a850c1e5bf/dask_geopandas/hilbert_distance.py#L112)
 and [[2]](https://github.com/tastatham/dask-geopandas/blob/a1eae6ceb8cd580f3d1f8240d22e52a850c1e5bf/dask_geopandas/morton_distance.py#L31) - although this is used in slightly different ways. 

Therefore it would make sense to centralize both `_continuous_to_discrete_coords` and (a generalised version of) `_distances_from_coordinates` (to work with both `_hilbert_distance` and `_morton_distance`). Does anyone have any sugestions where to centralize these functions?

Reply:
Can you make a note of it in the hilbetr follow-up issue (#85) and keep it as it is in this PR? We'll clean this later (I would even say that both `morton` and `hilbert` py files should go to `repartition` folder or something like that to keep the file structure clean.

Reply:
Updated issue #85 with a [new comment](https://github.com/geopandas/dask-geopandas/issues/85#issuecomment-894667314)